### PR TITLE
Add Stirling PDF

### PR DESCRIPTION
**Pull request by sharknoon**
# App Submission

### App name
Stirling PDF

### 256x256 SVG icon
icon

### Gallery images
Bildschirmfoto am 2024-05-29 um 10 21 29
Bildschirmfoto am 2024-05-29 um 10 22 41
Bildschirmfoto am 2024-05-29 um 10 22 33
Bildschirmfoto am 2024-05-29 um 10 22 23
Bildschirmfoto am 2024-05-29 um 10 22 18

### I have tested my app on:
- [ ] umbrelOS on a Raspberry Pi
- [ ] umbrelOS on an Umbrel Home
- [x] umbrelOS on Linux VM

### CODE REVIEW

**Comment by nmfretz** [comment]
Fantastic submission. Thanks @sharknoon! Just played around with Stirling PDF and I think this will be a permanent addition to my server.

We'll get gallery assets prepared and then go live to the app store.

File: stirling-pdf/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  app_proxy:
+    environment:
+      APP_HOST: stirling-pdf_app_1
+      APP_PORT: 8080
+      PROXY_AUTH_WHITELIST: "/api/*"
+  
+  app:
+    image: frooodle/s-pdf:0.25.0@sha256:2c67c213b1622abc613294f4883a7c9792d0bd1e566ab7960b48d0924f052b8a
+    # Stirling PDF does not work with a custom user
+    #user: "1000:1000"
```
Comment (nmfretz) [inline stirling-pdf/docker-compose.yml:13]: Thanks for checking this 🤝

**Comment by sharknoon** [comment]
You are very welcome 😀

**Comment by IROFI** [comment]
Good idea

**Comment by sharknoon** [comment]
You are very welcome 😀

**Comment by sharknoon** [comment]
@pimushkin Thank you for your feedback. I myself also run umbrelOS on the Raspberry Pi 5. Maybe the UI is wrong about the 50GB of free space? Please go to Settings -> Advanced Settings -> Terminal -> umbrelOS and run `df -h`. This shows you a list available space for each partition. Look for the partition that is mounted on `/`.

The second error comes from the Umbrel App Proxy into which have no influence over.

Thank you for reporting this issue :)

**Pull request merged**